var/spack/repos/builtin/packages/r-cardata/package.py

```diff
@@ -19,4 +19,4 @@ class RCardata(RPackage):
     version("3.0-2", sha256="3b5c4eff1cc1e456a5331084774503eaa06cf61fb7acf6b9e8a6bfabd5735494")
 
     depends_on("r@3.0:", type=("build", "run"))
-    depends_on("r@3.5:", type=("build", "run"), when="@3.0-4:")
+    depends_on("r@3.5.0:", type=("build", "run"), when="@3.0-4:")
```

Comment:
The version ranges are more explicit than necessary in some cases because these updates are in many cases [automatically derived](https://github.com/spack/spack/pull/45921) from the R package `DESCRIPTION` file. If that file specifies `>= 3.5.0`, then the package uses `@3.5.0:` (if it is being modified) otherwise, even if in the spack this is redundant.